packages/api/lib/response.js

```diff
@@ -226,16 +228,16 @@ async function getAuthorizationFailureResponse(params) {
   // Verify that the token exists in the DynamoDB Users table
   if (findUserResult.Count !== 1) {
     return buildAuthorizationFailureResponse({
-      error: 'invalid_token',
-      message: 'Invalid Authorization token'
+      message: 'User not authorized',
```

Comment:
We might want to define these strings somewhere

Reply:
Agreed.  I'd like to see those responses turned into classes, which would be the place to define that. I have a note to come back and refactor this response module later.